README.md

```diff
@@ -1,6 +1,199 @@
+Copying blob sha256:235ceb423e67f551977e1abac28d2eaae26e50a020e0972a28ca4b8d897178be
+Copying blob sha256:1b01ea1be8aca3fd95873b99083e97482c88374c6d648eef8031dc339c88ce2a
+Copying blob sha256:38f8e7472ea57428cc374eba8974cd1e340f69cb9fbba58bc036b8f9e6d52ab3
+Copying blob sha256:00f71969241bae02a4c5a8382fb32a6634e142e6cc780f6a9a21dbdceb77ce7f
+Copying blob sha256:b31b23df732d346b415e55c50f13261f5767ff01ad54abdf5dadc5a2cff46e01
+Copying blob sha256:2ac4b606308fae3b514bfadcb7b1a9b5666e8fa13849d568e1a85d2907ff7137
+Copying blob sha256:4f4fb700ef54461cfa02571ae0db9a0dc1e0cdb5577484a6d75e68dc38e8acc1
+Copying blob sha256:f0548e2cebadd91eb72e7ed4be37462f87973ebb11717fd739a28f6be63161c5
+Copying config sha256:4c881529d3e1d1dcf95c20645b2310571873bc0519278bd61c260fa3064ae6e6
+Writing manifest to image destination
+Storing signatures
+Release "cray-hms-test-development" has been upgraded. Happy Helming!
+NAME: cray-hms-test-development
+LAST DEPLOYED: Mon Mar 28 20:21:15 2022
+NAMESPACE: services
+STATUS: deployed
+REVISION: 4
+TEST SUITE: None
+NOTES:
+Installation info for chart cray-hms-test-development
+```
+
+## How to tell that the environment is working:
+
+NOTE: it may take a few minutes for this to be ready and to setup the persistent volume claim (PVC). 
+
+1. See if the deployment is operational:
+
+```
+ncn-m001:~/anieuwsma/hms-test # kubectl -n services get deployment  | grep cray-hms-test-development
+NAME                                              READY   UP-TO-DATE   AVAILABLE   AGE
+cray-hms-test-development                         1/1     1            1           51m
+```
+
+1. See if the pod is operational: 
+
+NOTE: you will need the exact pod name from this output
+
+```
+ncn-m001:~/anieuwsma/hms-test # kubectl -n services get pods | grep cray-hms-test-development
+NAME                                                              READY   STATUS                  RESTARTS   AGE
+cray-hms-test-development-6677f586dc-vp46b                        2/2     Running                 0          54m
+```
+
+## How to use the tools
+
+1. Exec into the pod and now you can use the tools
+
+```
+kubectl -n services exec -i -t cray-hms-test-development-6677f586dc-vp46b sh
+/src/app $
+```
+
+1. you need to modify the `tavern_global_config_integration_test.yaml` to look like this...
+
+```
+/src/app $ cat /src/libs/tavern_global_config_integration_test.yaml
+# This file contains the base common configurations for running pytest tavern tests.  It is statically generated,
+#  because we anticipate the same settings for all ct-test  containers that inherit from it.
+name: tavern_global_configuration #is this needed, used?
+description: common configuration for all tavern invocations
+variables:
+  verify: false #should ssl verification happen in tavern tests? its hard coded everywhere to false (partially because the PIT would complain)
+  base_url: http://httpbin.org/src/app
```

Comment:
This URL didn't work for me, it needed to be:

  base_url: http://httpbin.org/